src/core/random.js

```diff
@@ -0,0 +1,31 @@
+import { addrandomop } from './randomeval';
+
+/**
+ * rolldie
+ *
+ * Add a request to Roll a D6 die.
+ *
+ * The result is put into the given fieldname and is available
+ * on the next move.
+ *
+ * @param {...object} G - Game instance.
+ * @param {*} fieldname - Fieldname to put the random result into.
+ */
+export function rolldie(G, fieldname) {
```

Comment:
You'll need to import these in `packages/core.js` and `packages/main.js`. I have a mild preference to have these capitalized and camel-cased like `RollDie` and `Random`.

Reply:
Also, can you make this accept a second parameter for the die dimension? We can use a D6 if nothing is passed, but it would be nice to allow the user to override it.